helm/observability-bundle/templates/_helpers.tpl

```diff
@@ -28,14 +28,11 @@ When apps are created in the org namespace add a cluster prefix.
 Common labels
 */}}
 {{- define "labels.common" -}}
-app.kubernetes.io/name: {{ include "name" . | quote }}
-app.kubernetes.io/instance: {{ .Release.Name | quote }}
```

Comment:
This goes with https://github.com/giantswarm/giantswarm/issues/26818

Aligning with security bundle

Reply:
If we remove the label `app.kubernetes.io/name`, it can be an issue in other source code no, like here ?
https://github.com/giantswarm/prometheus-meta-operator/blob/cad4a29a00182119bc408fc0190dc3e07cddafe8/pkg/remotewriteutils/util.go#L35

Reply:
Well maybe we need to change the selector 😅 because if we set this label WE get some extra config :(